ee/ucp/admin/configure/set-orchestrator-type.md

```diff
@@ -100,6 +99,10 @@ Once a node is joined to the cluster, you can
 [change the orchestrator](#change-the-orchestrator-for-a-node) that schedules its
 workloads.
 
+> DTR in mixed mode
+>
+> The default and expected behavior for DTR nodes is to be in mixed orchestration. Additionally, if the DTR mode type is changed to Swarm only or Kubernetes only, reconciliation will revert the node back to mixed mode. This is expected behavior. 
```

Comment:
The first line states that it is the default and the expected behavior. Wondering if we can remove the 'expected behavior' in the first line to avoid repetition? Also suggest updating the last statement to 'This is the expected behavior`.